sdks/go/pkg/beam/io/healthcare/fhirio/fakes.go

```diff
@@ -0,0 +1,40 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fhirio
+
+import (
+	"net/http"
+)
+
+type fakeFhirStoreClient struct {
```

Comment:
Since this is only used for tests, I would put it in some file ending in `_test.go` so it doesn't get compiled into non-test binaries. Maybe in read_test.go for now, but even renaming this to fakes_test.go works.

Reply:
Updated to `fakes_test.go` since the code in here will be also used in the test files of my next upcoming PRs.